editor/editor_properties.cpp

```diff
@@ -2308,14 +2356,27 @@ void EditorPropertyResource::_resource_preview(const String &p_path, const Ref<T
 	}
 }
 
+void EditorPropertyResource::_script_created(Ref<Script> p_script) {
+
+	// Based partially upon editor/scene_tree_dock.cpp#L1628
```

Comment:
Same here, don't hardcode references to line numbers.

Reply:
Ditto for my comment above.